### PR TITLE
feat(vcl): Add extensions.vcl

### PR DESCRIPTION
**Pull request by kptdobe**
PR for https://github.com/adobe/helix-publish/issues/85

### CODE REVIEW

File: layouts/fastly/extensions.vcl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2018 Adobe. All rights reserved.
```
Comment (trieloff) [inline layouts/fastly/extensions.vcl]:
```suggestion
# Copyright 2019 Adobe. All rights reserved.
```

File: src/fastly/vcl.js [hunk 2]
```diff
@@ -26,6 +26,12 @@ async function init(fastly, version) {
   return fastly.setMainVCL(version, 'helix.vcl');
 }
 
+async function extensions(fastly, version) {
```
Comment (trieloff) [inline src/fastly/vcl.js]: How do you pass the customizations?

Comment (kptdobe) [reply src/fastly/vcl.js]: This will be part of https://github.com/adobe/helix-publish/issues/79 (clear separation of code ;) )

**Comment by kptdobe** [comment]
> Make sure to increase the number 6 to 7.

I assume you talk about a version number but where / when this increase should happen ?

Also, the tests are failing because of code coverage. But this seems to be inherited from `master`... why ? if code cover target is not reached and it is ok then we need to change the check to be green. Or... ?

**Comment by trieloff** [comment]
The only test that is failing on the CI is this:

Integration Test Test publish function locally - Test publish function locally
`/home/circleci/project/test/integration.js`
```
AssertionError [ERR_ASSERTION]: { body: { status: 'published', completed: 7 }, statusCode: 200 } deepStrictEqual { body: { status: 'published', completed: 6 }, statusCode: 200 }
    at Context.condit (test/integration.js:108:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

If you check the expected value, your tests should pass on the CI and your code coverage should be ok, too. Code coverage is lower when running locally without Fastly credentials because the tests that require these credentials get skipped.

**Comment by kptdobe** [comment]
Aggggrrr... I feature we cannot define subroutines in an external file! Need to find another way...

**Pull request merged**